Link error

Link error
```
../../lib/libocca-transpiler.so.17: undefined reference to `bool oklt::HandlerManager::registerBackendHandler<tl::expected<std::any, oklt::Error> (oklt::SessionStage&, clang::VarDecl const&, clang::Attr const&)>(oklt::TargetBackend, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, tl::expected<std::any, oklt::Error> (&)(oklt::SessionStage&, clang::VarDecl const&, clang::Attr const&))'
```

If in file `lib/core/handler_manager/backend_handler.h` at line 34 remove `inline` then links fine

--- a/lib/attributes/backend/openmp/exclusive.cpp
+++ b/lib/attributes/backend/openmp/exclusive.cpp
@@ -1,6 +1,6 @@
 #include "attributes/attribute_names.h"
 #include "attributes/utils/serial_subset/handle.h"
-#include "core/handler_manager/handler_manager.h"
+#include "core/handler_manager/backend_handler.h"
 
 #include <spdlog/spdlog.h>
 

--- a/lib/core/handler_manager/attr_handler.h
+++ b/lib/core/handler_manager/attr_handler.h
@@ -3,11 +3,12 @@
 #include "core/handler_manager/handler_manager.h"
 #include "util/type_traits.h"
 
+#include <oklt/util/string_utils.h>
+
 #include <clang/AST/ASTTypeTraits.h>
 #include <clang/AST/Attr.h>
 #include <clang/AST/Decl.h>
 #include <clang/AST/Stmt.h>
-#include <oklt/util/string_utils.h>
 #include <tl/expected.hpp>
 
 #include <any>

--- a/lib/core/handler_manager/implicid_handler.h
+++ b/lib/core/handler_manager/implicid_handler.h
@@ -3,9 +3,10 @@
 #include "core/handler_manager/handler_manager.h"
 #include "util/type_traits.h"
 
+#include <oklt/util/string_utils.h>
+
 #include <clang/AST/ASTTypeTraits.h>
 #include <clang/AST/Attr.h>
-#include <oklt/util/string_utils.h>
 #include <tl/expected.hpp>
 
 #include <any>

